src/utils/__tests__/math.spec.ts

```diff
@@ -35,16 +36,24 @@ describe('Test math functions', () => {
     );
   });
 
+  it('should add term2 to term1 and return correct value', () => {
+    const term1 = '1111111111';
+    const term2 = '0.00000001';
+    const result = '1111111111.00000001';
+
+    expect(addition(term1, term2)).toBe(result);
+  });
+
   it('should subtract decrement from number and return correct value', () => {
-    let value = 0.3;
-    let decrement = 0.1;
-    let result = 0.2;
+    let value = '0.3';
```

Comment:
I don't think it's correct that math lib return string as a result of number calculations

Reply:
We use big.js for some operations in math lib that returns string value. It's safe way to get correct values from calculations in some rare cases like 1111111111.00000008 (number will be rounded to 1111111111 by JS automatically). So I believe we should keep strings here.